lib/postman-sandbox.js

```diff
@@ -93,6 +93,7 @@ class PostmanSandbox extends UniversalVM {
             executionEventName = 'execution.result.' + id,
             executionTimeout = _.get(options, 'timeout', this.executionTimeout),
             cursor = _.clone(_.get(options, 'cursor', {})), // clone the cursor as it travels through IPC for mutation
+            requireFileCache = _.get(options, 'requireFileResolver'),
```

Comment:
TODO: 
- [ ] Update the option name here and in the consumers. `requireFileResolver` sounds like a function but it's a cache.

cc @aditya-baradwaj